GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -79,20 +79,11 @@ public override IEnumerable<EventListener> CreateTelemetryListeners(string provi
         {
             string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
 
-            ETWTelemetryEventListener etwListener = ETWTelemetryEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId);
-            if (etwListener != null)
+            TelemetryDaemonEventListener daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId);

```

Comment:
Are there any plans to support multiple listeners in the future?  If not, we could simplify the method signature to 
```
EventListener CreateTelemetryListener(string providerName, string enlistmentId, string mountId)
```

Reply:
Removed this entirely from `GVFSPlatform`.